checkOpenPMD_h5.py

```diff
@@ -808,6 +811,12 @@ def check_particles(f, iteration, v, pic) :
     # root attributes at "/"
     result_array = np.array([0, 0])
     result_array += check_root_attr(f, verbose, extension_pic)
+    
+    # Check for enabled extensions (After checking root attributes so we
+    # can detect if one is missing from the attribute)
+    valid, extensionIDs = get_attr(f, "openPMDextension")
+    if valid and (ext_list["ED-PIC"] & extensionIDs) == ext_list["ED-PIC"]:
```

Comment:
make this a own function with functionality described above (returns requiredExtensions to be passed to further functions)